apps.gen.go

```diff
@@ -0,0 +1,130 @@
+// Code generated by golang.org/x/tools/cmd/bundle. DO NOT EDIT.
+//   $ bundle -pkg godo -prefix  ./dev/dist/godo
+
+package godo
+
+import ()
+
+// AppDatabaseSpec struct for AppDatabaseSpec
+type AppDatabaseSpec struct {
+	Name     string                `json:"name"`
+	Engine   AppDatabaseSpecEngine `json:"engine,omitempty"`
+	Version  string                `json:"version,omitempty"`
+	Size     string                `json:"size,omitempty"`
+	NumNodes int64                 `json:"num_nodes,omitempty"`
+}
+
+// AppDatabaseSpecEngine the model 'AppDatabaseSpecEngine'
+type AppDatabaseSpecEngine string
+
+// List of AppDatabaseSpecEngine
+const (
+	APPDATABASESPECENGINE_UNSET AppDatabaseSpecEngine = "UNSET"
+	APPDATABASESPECENGINE_MYSQL AppDatabaseSpecEngine = "MYSQL"
+	APPDATABASESPECENGINE_PG    AppDatabaseSpecEngine = "PG"
+	APPDATABASESPECENGINE_REDIS AppDatabaseSpecEngine = "REDIS"
+)
+
+// AppRouteSpec struct for AppRouteSpec
+type AppRouteSpec struct {
+	Path string `json:"path,omitempty"`
+}
+
+// AppServiceSpec struct for AppServiceSpec
+type AppServiceSpec struct {
+	Name             string                  `json:"name"`
+	RunCommand       string                  `json:"run_command,omitempty"`
+	BuildCommand     string                  `json:"build_command,omitempty"`
+	HTTPPort         int64                   `json:"http_port,omitempty"`
+	DockerfilePath   string                  `json:"dockerfile_path,omitempty"`
+	Git              GitSourceSpec           `json:"git,omitempty"`
+	GitHub           GitHubSourceSpec        `json:"github,omitempty"`
+	Envs             []AppVariableDefinition `json:"envs,omitempty"`
+	InstanceSizeSlug string                  `json:"instance_size_slug,omitempty"`
+	// grpc-gateway's protoc-gen-swagger annonations only allow for a string-type default; encoding the default num type as a string openapi/validate to reject the spec. TODO(cbaker) - file bug / patch w/ upstream.
```

Comment:
Can we remove this comment?  I think we can remove it from the proto if that's what's required.